### PR TITLE
Added portfolio site to sites.yml

### PR DESCRIPTION
**Pull request by danielballoch**
## Description

Added my portfolio site (http://danielballoch.com) to the sites.yml file.

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -9657,3 +9657,18 @@
   built_by: Kashaf S
   built_by_url: https://www.linkedin.com/in/kashaf-shaikh-925117178
   featured: false
+  - title: Daniel Balloch
```
Comment (laurieontech) [inline docs/sites.yml]:
```suggestion
- title: Daniel Balloch
```

**Comment by laurieontech** [comment]
Thanks for the PR! Looks like there was an errant space that messed with the yaml, so made a recommendation to fix that.
One other thing I'd note is that we'd love the README for your site to include instructions for running it. Something like `npm install`, `gatsby develop`, etc.
But overall looks great and with a couple edits we can get this merged.

**Comment by danielballoch** [comment]
Thanks for that! 
Just added a install instructions as suggested (README and description).
Do I need to make any other changes or is this okay?

**Comment by laurieontech** [comment]
Nope, this is good to go! Thanks for making the changes.

**Pull request merged**